config/dev.exs

```diff
@@ -1,7 +1,7 @@
 use Mix.Config
 
 config :kafka_ex,
-  brokers: [{"ip-10-0-0-49.ec2.internal", 9092}, {"ip-10-0-0-248.ec2.internal", 9092}],
+  brokers: [{"ip-10-0-0-166.ec2.internal", 9092}],
```

Comment:
I couldn't get that to work when I tried it with Causality, but maybe the Elixir library can work with those hostnames.